src/components/ui/ToggleGroup/ToggleGroup.tsx

```
@@ -5,7 +5,68 @@ import ToggleItem from './fragments/ToggleItem';
 
 const COMPONENT_NAME = 'ToggleGroup';
 
-const ToggleGroup = ({ type = 'single', items = [] as { value: any; label: any }[], color = '' }) => {
+/**
+ * ToggleGroup Item interface
+ * @typedef ToggleItem
+ */
+type ToggleItem = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ToggleItem'. Consider to delete it or rename it.

'ToggleItem' is defined here:


(lint/suspicious/noRedeclare)